frontend: Add email illustrations to static/images/emails.

vaidap opened the pull request:
Will allow me to test #5592 with the actual path to the images hosted on `chat.zulip.org` rather than an imgur link.

smarx commented:
*Automated message from Dropbox CLA bot*

@vaidap, it looks like you've already signed the Dropbox CLA. Thanks!

timabbott commented:
@vaidap can you put these in `static/images/emails/` or something?  `static/generated` is a special directory where files generated by scripts go.

vaidap commented:
made the change!

rishig commented:
cool, thanks @vaidap! A few minor comments: 
* Let's change the name of followup21.png to day2_1.png, and similarly for the others
* For followup22: Let's change the fish texts to "design mockup", keep the russian one as is, "acme burgers", and "Bug 345". Do those work (in particular, is "Bug 345" long enough?)
* For followup23: Let's change the text to "Where should we go to lunch today?" Is that long enough?
* For logo.png, let's name it email_logo.png, and also make it much smaller. It should be decoration I think, rather than taking so much visual attention / height.

vaidap commented:
Updated the illustrations,
* I think "Bug 345 works" fine, i updated the size of the container to make it work.
* "Where should we go to lunch today?" was a bit short so also adjusted the width

rishig commented:
Thanks Vaida! @timabbott, ready to merge from my end.

timabbott commented:
Merged, thanks @vaidap!  This reminds me: Do you have original SVG files used to generate these PNGs?  We generally put that sort of thing under `static/assets/` so that we have them nicely backed up in Git.

vaidap commented:
@timabbott yes, I keep all the source files-- opened #6258! Was also wondering if other illustrations that have been implemented should also be moved from the Dropbox to here?

rishig commented:
yup! It would be great to have all the svgs for any pngs that are used :).

vaidap commented:
Opened #6276 -- it made more sense to just change the feature images to SVGs rather than duplicate them in the assets folder. Apart from those the other images are still in flux i believe!
